runtimes/datascience/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -351,7 +351,10 @@ fi
 # Install Python packages from pylock.toml
 COPY ${DATASCIENCE_SOURCE_CODE}/pylock.toml ./
 # Copy Elyra dependencies for air-gapped enviroment
-COPY ${DATASCIENCE_SOURCE_CODE}/utils ./utils/
+COPY --chown=1001 ${DATASCIENCE_SOURCE_CODE}/utils ./utils/
+# Download Elyra Bootstrapper
+RUN curl -L https://raw.githubusercontent.com/opendatahub-io/elyra/refs/tags/v4.3.1/elyra/kfp/bootstrapper.py \
```

Comment:
Can we read the version dynamically from the installed odh-elyra package instead of hardcoding `4.3.1`? Otherwise, we'll have to remember to update this version every time we update the package.

Reply:
The only problem is that I don't have this information in `runtime` images because Elyra is not installed there, only the runtime image dependencies are installed.